### ISSUE TITLE
Error on creating the virtual drive

### ISSUE THREAD

**Issue by freskhu** [itsToggle/plex_debrid#622]
root@plex:~# ./rclone-linux mount my-remote: Y: --dir-cache-time 10s
<5>NOTICE: realdebrid root '': --vfs-cache-mode writes or full is recommended for this remote as it can't stream
mount helper error: fusermount: fuse device not found, try 'modprobe fuse' first
Fatal error: failed to mount FUSE fs: fusermount: exit status 1

**Comment by JHarding86** [comment]
Did you try appending `--vfs-cache-mode writes` to your mount command?

**Comment by freskhu** [comment]
yes but the problem is on fusemount, 
root@plex:~# modprobe fuse
modprobe: FATAL: Module fuse not found in directory /lib/modules/6.5.11-7-pve

**Comment by freskhu** [comment]
I tired so many things and i am not able to surpass this
root@plex:/var/lib/plexmediaserver# ./rclone-linux mount realdebrid: torrents --dir-cache-time 10s --allow-other --vfs-cache-mode writes
mount helper error: fusermount: fuse device not found, try 'modprobe fuse' first
Fatal error: failed to mount FUSE fs: fusermount: exit status 1

**Comment by JHarding86** [comment]
Have you tried to install fuse?

'sudo apt install fuse'

And libfuse?

'sudo apt install libfuse2'

**Comment by freskhu** [comment]
I managed to solve it...
I am running into LXC proxmox and it is solved with the command mknod -m 666 /dev/fuse c 10 229

**Comment by dreulavelle** [comment]
> I managed to solve it... I am running into LXC proxmox and it is solved with the command mknod -m 666 /dev/fuse c 10 229

Is that an Alpine LXC? I been trying to figure that one out myself lol. I just switched to a Debian LXC instead and it solved it. 

Thanks for the tip!

**Comment by freskhu** [comment]
its the script from tteck

**Comment by slunat** [comment]
> I managed to solve it... I am running into LXC proxmox and it is solved with the command mknod -m 666 /dev/fuse c 10 229

I just tried this because i was getting the same issue, im also running in a proxmox container. When i run the rclone mount after doing this though, it just hangs. No errors, but it doesnt give me the mesage to say its succesfully either and i cant enter a new command because its stuck processing.

**Comment by freskhu** [comment]
> > I managed to solve it... I am running into LXC proxmox and it is solved with the command mknod -m 666 /dev/fuse c 10 229
> 
> I just tried this because i was getting the same issue, im also running in a proxmox container. When i run the rclone mount after doing this though, it just hangs. No errors, but it doesnt give me the mesage to say its succesfully either and i cant enter a new command because its stuck processing.

it means it worked

**Comment by slunat** [comment]
> it means it worked

Yeah i realised afterwards it was actually running. The guide mentioned a message would show up saying it was successful so I assumed it wasnt running. Also for me I was able to get around this issue by just enabling the Fuse feature in the options for the LXC.
